electroncash_plugins/fusion/fusion.py

```diff
@@ -459,11 +459,15 @@ def run(self):
 
             self.status = ('complete', 'time_wait')
 
-            # wait up to a minute before unfreezing coins
+            # The server has told us the fusion is complete but we might not
+            # have seen the tx show up in our wallets. So we can't unfreeze
+            # coins or unreserve addresses yet, else they might be used in
+            # another fusion. Wait up to a minute for this to happen.
+            wallets = set(self.source_wallet_info.keys()) + {self.target_wallet}
```

Comment:
You cannot add sets in python like this.  This will raise a TypeError.  One needs to do `|` for sets.

I suggest actually you use lists here -- e.g. `list(self.source_wakket_info) + [self.target_wallet]`.. faster than set, and also supports `+`.  If you do use `set`, use `|`.

Reply:
ah, that's what I get for not full testing this :D

Reply:
please to fully testing this :)